src/app/aether.component.ts

```diff
@@ -83,6 +82,7 @@ export class AetherComponent implements OnInit, OnDestroy {
             // When no auth is used just open Web socket and accept everything
             this.socketService.connect();
         }
+        this.enterpriseService.loadTargets();
```

Comment:
in an Auth situation it will try to do this before the connection has succeeded and will fail - it should work in both scenarios

Reply:
although /targets might not be protected by Auth presently - it should be though